test/boilerplate_suggestors/advanced_props_and_state_class_migrator_test.dart

```diff
@@ -387,7 +414,6 @@ void advancedPropsAndStateClassMigratorTestHelper({
               'ADifferentPropsClass': 'ADifferentPropsClassMixin',
             },
           )(
-            expectedPatchCount: 6,
```

Comment:
I removed most of the `expectedPatchCount` assertions in this file, since they don't provide much value (patches are already tested via `expectedOutput`) and are a pain to update.

I left the ones where the value was `0`/1`.